python/ray/util/placement_group.py

```diff
@@ -147,6 +148,12 @@ def placement_group(
             will fate share with its creator and will be deleted once its
             creator is dead, or "detached", which means the placement group
             will live as a global object independent of the creator.
+        max_cpu_fraction_per_node: The maximum fraction of CPU cores this placement
```

Comment:
let's keep this private?

Reply:
also, as we discussed this is confusing if multiple placement group specify `max_cpu_fraction_per_node`. can we clarify in comment, or we rename the parameter to be more explicit.

Reply:
done